flexmeasures/ui/__init__.py

```diff
@@ -28,6 +28,7 @@
     parse_config_entry_by_account_roles,
     find_first_applicable_config_entry,
 )
+from flexmeasures.ui.utils.color_defaults import get_color_settings
```

Comment:
Because we're still in the `__init__` of this package, I think it is safer to import this within `register_at()`, as we've done it with other packages there as well.

Reply:
ok i can do that